src/ParserInterface.php

```diff
@@ -12,4 +12,9 @@
      * @return array
      */
     public function parse(array &$menu, array $config);
+
+    /**
+     * @return boolean
+     */
+    public function hasSelected();
 }
```

Comment:
Not that it really matters, but it probably doesn't need to be a part of the interface since as its not really a requirement to the core functionality of Parsing something (like the other functions in the current class)